From 7637b20149d17029551e744daa8985a3faea2bc8 Mon Sep 17 00:00:00 2001
From: LN Liberda <lauren@selfisekai.rocks>
Date: Thu, 22 Jan 2026 10:32:30 +0100
Subject: [PATCH] Allow any wasm-bindgen version

We need it to work for building our packages, which will have arbitrary versions specified
---
 src/install/mod.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/install/mod.rs b/src/install/mod.rs
index a5c963f..d977483 100644
--- a/src/install/mod.rs
+++ b/src/install/mod.rs
@@ -102,7 +102,7 @@ pub fn check_version(tool: &Tool, path: &Path, expected_version: &str) -> Result
         "Checking installed `{}` version == expected version: {} == {}",
         tool, v, &expected_version
     );
-    Ok(v == expected_version)
+    Ok(true)
 }
 
 /// Fetches the version of a CLI tool
